docs/Advanced-Topics-Entities.md

```diff
@@ -67,11 +67,11 @@ const contentState = editorState.getCurrentContent();
 const contentStateWithEntity = contentState.createEntity(
   'LINK',
   'MUTABLE',
-  {href: 'http://www.zombo.com'}
+  {url: 'http://www.zombo.com'}
```

Comment:
You should also change line 89 to use `url` instead of `href` for the sake of consistency.